src/main/webapp/javascript/user-page.js

```diff
@@ -7,16 +8,65 @@ async function loadUserRecipes() {
   const response = await fetch('/list-private-recipes');
   const recipes = await response.json();
 
+  let recipesDiv = document.getElementById("user-recipes");
+  recipesDiv.innerHTML = "";
+  
+  let size = 0;
   if (Object.keys(recipes)) {
-    const recipesElement = document.getElementById('user-recipes');
-    recipesElement.innerText = "";
     for (let key of Object.keys(recipes)) {
+        size++;
         let value = recipes[key];
-        // createUserRecipeCard(value)
+        createUserRecipeCard(value);
+    }
+    if(size == 0){
```

Comment:
nit: space after if and before {

also, you may be able to check Object.keys(recipes).length to get the size, instead of keeping track of a size variable.

Reply:
oh okay that makes sense 
I tried to get the size of recipes by itself and it didn't work which is why I did the size thing